website/storybook/1-components/View/ViewScreen.js

```diff
@@ -125,13 +127,28 @@ const ViewScreen = () => (
       <DocItem
         name="onLayout"
         typeInfo="?function"
-        description={
+        description={[
           <AppText>
             Invoked on mount and layout changes with{' '}
             <Code>{'{ nativeEvent: { layout: { x, y, width, height } } }'}</Code>, where{' '}
             <Code>x</Code> and <Code>y</Code> are the offsets from the parent node.
+          </AppText>,
+          <AppText>
+            NOTE: Behind the hood React Native for Web uses <Code>ResizeObserver</Code> and doesn't
```

Comment:
Please could you include `ResizeObserver` and a link to the polyfill in the list of recommended polyfills in the "Getting started" docs

Reply:
will do